### DISCUSSION TITLE
Add a collapse button to the left sidebar

### DISCUSSION THREAD

**Discussion by Sunny19191** [web-infra-dev/rspress#1217] [Ideas]
像Docusaurus一样，增加侧边栏收起功能
![收起按钮](https://github.com/web-infra-dev/rspress/assets/171238222/ba4e8f41-e444-4479-aa00-5ab0a589ac4f)

**Comment by Timeless0911** [comment]
The same as https://github.com/web-infra-dev/rspress/discussions/1214#discussioncomment-9892813.

We may consider adding an option to control whether this collapse button exists in the future, PR is welcome.
